### ISSUE TITLE
Ultra wide support for inventory mode

### ISSUE THREAD

**Issue by takashi1kun** [bennybroseph/AutoHotKey_Scripts#2]
There is no support for inventory mode in ultra-wide resolutions like 2560x1080, changing the numbers in the inventory options custom scaling doesn't seem to work, I have tried to set them to:

### 1. Directly putting the aspect ratio:
Custom_Scaling_Width 21 
Custom_Scaling_Height 9

### 2. Putting in width the result of dividing 2560/1920 in width
Custom_Scaling_Width 1.3333
Custom_Scaling_Height 1

###  3. Directly putting my resolution
Custom_Scaling_Width 2560
Custom_Scaling_Height 1080

Nothing seems to work if I put the either the first or third the mouse goes to very wrong places, and in the second option the scripts get bugged and I have to restart it, it doesn't seem to take floats too well

**Comment by takashi1kun** [comment]
The map button also doesn't seem to show up for some reason

**Comment by takashi1kun** [comment]
Okay solved, it is a little weird how you have to configure this but this would be more or less the way:

1. You have to use both custom scaling and offset width to configure it.
2. For custom scaling width you have to divide 1920 between the resolution that you want, for example, for a 2560 one you make 1920/2560, in this case, 0.75, so you put 0.75 there, also be wary and not use the form if you are in a country that treats the comma as the separator instead of the dot, because it will put a comma in the config file instead of a dot.
3. Then you have to configure the width offset, in this case, you have to make a subtraction, you want to subtract 1920 from your chosen resolution, in my case, it would be 2560-1920= 640, so you put this result in the offset width

@bennybroseph Being that the case that these numbers can be calculated methodically, wouldn't it be better to allow for a boolean that you can toggle true or false and if true it just takes your resolution from Base_Resolution_Width  and does this math for you?

**Comment by adanyel** [comment]
This is really helpful for all the ones who has a 19:10 monitor, thanks @takashi1kun.
Do you manage to solve also the Overlay buttons? I was trying with some numbers with no luck.

**Comment by bennybroseph** [comment]
Yeah, probably an oversight on my part. I designed the program with as much customizability as I could think of, so doing something automatically probably didn't occur or never came up. I think at some point someone said they played in windowed mode or in fullscreen with a low resolution or something, idk. Adding a small on/off for it would work all the same since you could simply turn it off for weird edge cases where you need to set it yourself.
